pkg/models/querybuilder_sql.go

```diff
@@ -37,22 +38,18 @@ func handleStringCriterion(column string, value *StringCriterionInput, query *da
 	}
 }
 
-func insertObject(tx *sqlx.Tx, table string, object interface{}) (int64, error) {
+func insertObject(tx *sqlx.Tx, table string, object interface{}) error {
 	ensureTx(tx)
 	fields, values := SQLGenKeysCreate(object)
 
-	result, err := tx.NamedExec(
+	_, err := tx.NamedExec(
 		`INSERT INTO `+table+` (`+fields+`)
 				VALUES (`+values+`)
 		`,
 		object,
 	)
 
-	if err != nil {
-		return 0, err
-	}
-
-	return result.LastInsertId()
+	return err
 }
 
 func insertJoins(tx *sqlx.Tx, table string, joins interface{}) error {
```

Comment:
Since `insertObject` no longer returns the id, I think we can rename this to `insertObjects`.

Reply:
It still only inserts a single object so I'm not quite sure why we would change it?

Reply:
This refers to line 55 - to rename `insertJoins` to `insertObjects`.

Reply:
Oh, gotcha, the diff confused me. I've updated the name.